pgmq-extension/sql/pgmq.sql

```diff
@@ -687,33 +687,33 @@ BEGIN
 
   EXECUTE FORMAT(
     $QUERY$
-    CREATE TABLE IF NOT EXISTS pgmq.q_%s (
+    CREATE TABLE IF NOT EXISTS pgmq.%I (
         msg_id BIGINT GENERATED ALWAYS AS IDENTITY,
         read_ct INT DEFAULT 0 NOT NULL,
         enqueued_at TIMESTAMP WITH TIME ZONE DEFAULT now() NOT NULL,
         vt TIMESTAMP WITH TIME ZONE NOT NULL,
         message JSONB
     ) PARTITION BY RANGE (%s)
     $QUERY$,
-    queue_name, partition_col
+    'q_' || queue_name, partition_col
   );
 
-  IF NOT pgmq._belongs_to_pgmq(FORMAT('q_%s', queue_name)) THEN
-      EXECUTE FORMAT('ALTER EXTENSION pgmq ADD TABLE pgmq.q_%s', queue_name);
+  IF NOT pgmq._belongs_to_pgmq(FORMAT('%I', 'q_' || queue_name)) THEN
+      EXECUTE FORMAT('ALTER EXTENSION pgmq ADD TABLE pgmq.%I', 'q_' || queue_name);
   END IF;
 
   EXECUTE FORMAT(
     $QUERY$
-    SELECT public.create_parent('pgmq.q_%s', '%s', 'native', '%s');
+    SELECT public.create_parent('pgmq.%I', '%s', 'native', '%s');
     $QUERY$,
-    queue_name, partition_col, partition_interval
+    'q_' || queue_name, partition_col, partition_interval
   );
```

Comment:
Arguments to a function should be passed as string literals, not identifiers, right?

```SQL
    SELECT public.create_parent('pgmq.%I', %L, 'native', %L);
```

Honestly, though, you can just call functions:

```sql
    PERFORM public.create_parent(
        'pgmq.' || quote_ident('q_' || queue_name),
        partition_col, partition_interval
    );
````

Reply:
good idea!